fastlane/Fastfile

```diff
@@ -268,7 +265,9 @@ def check_no_git_tag_exists(version_number)
   end
 end
 
-def update_sample_podfile_lock_with_retry(pod_name)
+private_lane :update_sample_podfile_lock_with_retry do |options|
```

Comment:
Just curious, any reason to change to use a private_lane?

Reply:
yes, this https://github.com/RevenueCat/react-native-purchases/pull/954#discussion_r1537766221

lanes are methods which you pass a block to, the block (or closure) is able to see the scope of the thing that it was called from which is why it can see those variables defined at the top of the file but a method can't